datasource/src/ConfigEditor.tsx

```diff
@@ -61,7 +61,7 @@ export class ConfigEditor extends PureComponent<Props, State> {
             checked={!!jsonData.ssl}
           />
           <FormField
```

Comment:
I would suggest replacing this legacy form with proper react components. I drafted something here, but didn't have a chance to test it yet: https://github.com/tkurki/signalk-grafana/compare/master...plektra:signalk-grafana:master

Reply:
Just been lazy, as things have been working. PRs are welcome, but before refactoring much I'd like to know if the datasource is working for you as is?

Reply:
Yes it is working nicely. However "Explore" shows the initial graph with only data points, not interconnected lines. 
<img width="1305" alt="image" src="https://github.com/user-attachments/assets/f493e67a-cf24-4375-aef2-10d64e76b50c">